hw-model/c-binding/build.rs

```diff
@@ -0,0 +1,31 @@
+// Licensed under the Apache-2.0 license
+
+extern crate cbindgen;
+
+use std::path::PathBuf;
+use std::{env, str::FromStr};
+
+fn main() {
+    // Get Crate dir
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    // Generate Config
+    let config = cbindgen::Config {
+        header: Some(String::from_str("// Licensed under the Apache-2.0 license").unwrap()),
+        language: cbindgen::Language::C,
+        ..Default::default()
+    };
+
+    // Generate Output file
+    let out_file = PathBuf::from(&crate_dir)
```

Comment:
So far caliptra-sw has been following the pattern of checking generated code into the repo (like the caliptra-registers crate), and then enforcing in CI that the generated code is up to date. Three reasons:

- Code reviewers can easily tell during code review what the generated code looks like, and how a particular PR affects the generated code, which isn't always obvious.

- When discussing the generated code in Slack or email, we can paste links directly to the generated code in GitHub.

- The build is faster, as cbindgen and its dependencies don't normally need to be built.

We can leave this as-is for now, but I may revisit this in a future PR.

Reply:
I can take care myself in a separate (and smaller PR).